x-pack/plugins/security_solution/server/endpoint/routes/resolver/queries/events.ts

```diff
@@ -86,6 +86,7 @@ export class EventsQuery extends BaseResolverQuery {
     return {
       body: this.query(filters),
       index: this.indexPatterns,
+      allow_partial_search_results: true,
```

Comment:
during testing it seemed that some integrations did not have a mapping for event.id, and so had to add this for queries to still return results.

Reply:
Yea, it shouldn't as we're searching on the doc id.